modules/xfem/src/utils/XFEMParallel.C

```diff
@@ -0,0 +1,66 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+
+#include "XFEMParallel.h"
+#include "MooseError.h"
+
+namespace XFEMUtils
+{
+
+void
+allgatherStringBuffers(const Parallel::Communicator & comm,
+                       std::string & send_buffer,
+                       std::vector<std::string> & recv_buffers)
+{
+  // serial case
+  if (comm.size() < 2)
```

Comment:
The libMesh version of all gather handles this case too.

Reply:
This is awesome!  I lifted this code from @dschwen at his suggestion because he said there were some size limitations with the libmesh wrapped version. It turns out that he had actually added them to libmesh, but forgot that he did so. I will remove this code and call the libmesh version instead.

Reply:
That's what senility does 😉